src/biogeochem/CNFireLi2024Mod.F90

```diff
@@ -605,19 +600,24 @@ subroutine CNFireArea (this, bounds, num_soilc, filter_soilc, num_soilp, filter_
               end if
               lh       = pot_hmn_ign_counts_alpha*6.8_r8*hdmlf**(0.43_r8)/30._r8/24._r8
               fs       = 1._r8-(0.01_r8+0.98_r8*exp(-0.025_r8*hdmlf))
-              ig       = (lh+this%forc_lnfm(g)/(5.16_r8+2.16_r8* &
-                     cos(SHR_CONST_PI/180._r8*3*min(60._r8,abs(grc%latdeg(g)))))* &
-                         cnfire_params%ignition_efficiency)*(1._r8-fs)*(1._r8-cropf_col(c))
+              if (trotr1_col(c)+trotr2_col(c)<=0.6_r8) then
```

Comment:
This is NOT something to work on now. But, as pointed out by "8" in #12 there's a bunch of magic numbers here that should be made into parameters. I suspect a lot of them are shared between all of the Li fire method versions. This would be a nice code health improvement to make for the future.

Reply:
This is another point that would be good to have a comment to explain what's going on...

Reply:
Yup, magic numbers are one of the big things I'm planning to fix in the eventual refactor. Added this to my new issue #2714.